include/Switch.hpp

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include "NetworkDevice.hpp"
+#include "Packet.hpp"
+
+class Switch: public NetworkDevice {
+public:
+    Switch(double processing_delay_ns);
```

Comment:
I don't know how to write about this correctly, but I want to ask you. Up to this point, all the numbers we used were integers. So, why did you decide to calculate the delays in double